src/miniscript/context.rs

```diff
@@ -739,9 +739,9 @@ impl ScriptContext for BareCtx {
 
     fn pk_len<Pk: MiniscriptKey>(pk: &Pk) -> usize {
         if pk.is_uncompressed() {
-            65
+            66
         } else {
-            33
+            34
```

Comment:
nit: Would it make sense to have `33 + 1` instead? This way we can explicitly show `OP_PUSH...` and `<pk>`.